tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -35,6 +35,11 @@
     file: ospdo.yaml
   when: ospdo_src| bool
 
+- name: Override netconfig_networks if netconfig_networks_override is defined
+  ansible.builtin.set_fact:
+    netconfig_networks: "{{ netconfig_networks_override }}"
```

Comment:
I think the same can be achived by setting netconfig_networks: {{ netconfig_networks_override }} in job definition in vars, without setting additional fact.